<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckOnWork</class>
 <widget class="QWidget" name="CheckOnWork">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_cam">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>461</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>40</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>NAME</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_id">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>120</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_class">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>190</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CLASS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_work">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>380</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>开始识别</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
